Dockerfile

```diff
@@ -29,6 +31,35 @@ RUN \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* && \
     pip3 install gcovr
 
+# installing mongo drivers
+RUN \
+    echo "Installing mongo-c-driver" && \
+    apt-get -qq update && \
+    apt-get install -y \
+        pkg-config \
+        libssl-dev \
+        libsasl2-dev \
+        wget \
+    && \
+    wget https://github.com/mongodb/mongo-c-driver/releases/download/1.9.5/mongo-c-driver-1.9.5.tar.gz && \
+    tar xzf mongo-c-driver-1.9.5.tar.gz && \
+    cd mongo-c-driver-1.9.5 && \
+    ./configure --disable-automatic-init-and-cleanup --enable-static && \
+    make && \
+    make install && \
+    cd .. && \
+    rm -rf mongo-c-driver-1.9.5 && \
+    echo "Installing mongo-cxx-driver" && \
+    git clone https://github.com/mongodb/mongo-cxx-driver.git --branch releases/v3.2 --depth 1 && \
+    cd mongo-cxx-driver/build && \
+    cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr/local .. && \
+    make EP_mnmlstc_core && \
+    make && \
+    make install && \
+    cd ../.. && \
+    rm -rf mongo-cxx-driver
+# end
+
```

Comment:
Mongo should be an optional configration:

- Please add an additional dockerfile-mongo. 
- And implement installing of mongo driver into it.
- Also add an option to CMakeLists.txt.